Obsidian.API/CommandFramework/Attributes/RequirePermissionAttribute.cs

```diff
@@ -6,21 +6,22 @@ namespace Obsidian.Commands
 {
     public class RequirePermissionAttribute : BaseExecutionCheckAttribute
     {
-        string permission = "";
-        bool op = true;
+        private string[] permissions;
+        private bool op;
 
-        public RequirePermissionAttribute(string permission = "", bool op = true)
+        public RequirePermissionAttribute(bool op = false, params string[] permissions)
         {
-
+            this.permissions = permissions;
+            this.op = op;
         }
 
         public override async Task<bool> RunChecksAsync(ObsidianContext ctx)
         {
             if (this.op && ctx.Player.IsOperator)
                 return true;
 
-            if (this.permission != "")
-                return await ctx.Player.HasPermission(permission.ToLower());
+            if (this.permissions != null)
+                return await ctx.Player.HasAnyPermission(permissions);
```

Comment:
Wouldn't we expect that all permissions match? Maybe an enum for telling Obsidian if you want to match All, Any or None permissions.
I won't require to add this, just document it somewhere using xmldocs.

Reply:
Craft is right, the attribute should check ALL perms by default, not ANY.

Reply:
Yep, that's actually a good idea.